packages/cli/src/main.ts

```diff
@@ -69,7 +69,7 @@ export const main = () => {
       new Option(
         '--dotEnvPath <value>',
         'path to the `.env` file which stores the environment variable `STORYBLOK_PERSONAL_ACCESS_TOKEN`',
-      ).default('.'),
+      ),
```

Comment:
When the path is not given, it automatically reads `.env` and `.env.local` from the current working directory.

If given, it loads the specific file.

Reply:
Should we also provide an error message in case the .env is not found? Something like '.env and .env.local were not found, please try running this command again and add ---dotEnvPath to specify the location of ....'